examples/chart/teleport-kube-agent/README.md

```diff
@@ -181,9 +181,40 @@ $ helm install teleport-kube-agent . \
   --set roles=db \
   --set proxyAddr=${PROXY_ENDPOINT?} \
   --set authToken=${JOIN_TOKEN?} \
-  --set "awsDatabases[0].types=${DB_TYPES?}" \
-  --set "awsDatabases[0].regions=${DB_REGIONS?}" \
-  --set "awsDatabases[0].tags=${DB_TAGS?}"
+  --set "awsDatabases[0].types={${DB_TYPES?}}" \
+  --set "awsDatabases[0].regions={${DB_REGIONS?}}" \
+  --set "awsDatabases[0].tags.${DB_TAG_KEY?}={${DB_TAG_VALUES?}}"
+```
+
+### Auto-discovery mode (Azure)
+
+To use Teleport database access in Azure database auto-discovery mode, you will also need:
+- the database types you are attempting to auto-discover (`$DB_TYPES`)
+- the Azure resource tags if you want to target only certain databases (`$DB_TAG_KEY`, `$DB_TAG_VALUES`)
+
+You can optionally specify:
+- the Azure subscription(s) to auto-discover in (`$DB_SUBSCRIPTIONS`)
+- the Azure region(s) to auto-discover in (`$DB_REGIONS`)
+- the Azure resource-group(s) to auto-discover in (`$DB_RESOURCE_GROUPS`)
+
+The default for each of these optional settings is `*`, which will auto-discover in all
+subscriptions, regions, or resource groups accessible by the Teleport service
+principal in Azure.
+
+To install the agent in Azure database auto-discovery mode, run:
+
+```sh
+$ helm install teleport-kube-agent . \
+  --create-namespace \
+  --namespace teleport \
+  --set roles=db \
+  --set proxyAddr=${PROXY_ENDPOINT?} \
+  --set authToken=${JOIN_TOKEN?} \
+  --set "azureDatabases[0].types={${DB_TYPES?}}" \
+  --set "azureDatabases[0].tags.${DB_TAG_KEY?}={${DB_TAG_VALUES?}}" \
+  --set "azureDatabases[0].subscriptions={${DB_SUBSCRIPTIONS?}}" \
+  --set "azureDatabases[0].regions={${DB_REGIONS?}}" \
+  --set "azureDatabases[0].resource_groups={${DB_RESOURCE_GROUPS?}}"
 ```
```

Comment:
If I follow the README I'll end up with a non working deployment because this does not mentions azure authentication (so does the AWS one). Maybe it would be better to get rid of the command in the README and link to the reference as it already explains how this parameter works and what is required for the pod to authenticate to azure.